docs/storage.md

```diff
@@ -78,7 +77,7 @@ config:
   access_key: ""
   insecure: false
   signature_version2: false
-  secret_key: ""
+  secret_key: null
```

Comment:
I guess we can eliminate this changes by initializing this field by an empty string.